libs/payments/cart/src/lib/cart.utils.ts

```diff
@@ -26,3 +57,7 @@ export function stripeErrorToErrorReasonId(
       return CartErrorReasonId.Unknown;
   }
 }
+
+export function handleEligibilityStatus(eligibility: EligibilityStatus) {
+  return cartEligibilityDetailsMap[eligibility];
+}
```

Comment:
Does this function need to exist now that it's just a map lookup?

Reply:
Good catch. I was going by the original AC but since making a map (more readable), this doesn't seem necessary anymore. I'll update.